### DISCUSSION TITLE
Dumb question: is it posssible to use rockchip-6.1 overlay files in mainline-6.7 build

### DISCUSSION THREAD

**Discussion by nobara256** [Joshua-Riek/ubuntu-rockchip#630] [Q&A] [answered]
Hello again, I am using OPi5 Plus as router with last mainline-6.7 build for months and it runs surprisingly well with intel WiFi cards. Now I want some I2C and SPI devices to interface with but the build lacks overlay files. Is it possible to use the files from rockchip builds or get a mainline build with the overlays?

Thanks in advance. Appreciate the effort behind this awesome project.

**Comment by Joshua-Riek** [comment, marked as answer]
No, they need to be compiled against the currently used kernel. I'm spending a lot of time on the 6.1 kernel right now, but when it's stable I will start improving mainline Linux again.
